pkg/providers/launchtemplate/suite_test.go

```diff
@@ -204,6 +217,7 @@ var _ = Describe("LaunchTemplate Provider", func() {
 	})
 	It("should default to a generated launch template", func() {
 		ExpectApplied(ctx, env.Client, nodePool, nodeClass)
+		ExpectReconcileSucceeded(ctx, statusController, client.ObjectKeyFromObject(nodeClass))
```

Comment:
So we don't have to reconcile, should we just define status in the beforeEach and set the status to match? I'm fine either way, but just curious what you think about that option vs. directly calling the status controller reconcile